platform/platform-configuration/src/main/java/org/codice/ddf/configuration/ConfigurationManager.java

```diff
@@ -110,9 +110,9 @@
   private static Map<String, String> propertyMapping = new HashMap<>();
 
   static {
-    propertyMapping.put(PROTOCOL, SystemBaseUrl.PROTOCOL);
-    propertyMapping.put(HOST, SystemBaseUrl.HOST);
-    propertyMapping.put(PORT, SystemBaseUrl.PORT);
+    propertyMapping.put(PROTOCOL, SystemBaseUrl.EXTERNAL_PROTOCOL);
```

Comment:
❓ Are we sure all the users of platform-config are external?

Reply:
Good question, I'm not sure.